### ISSUE TITLE
comparisonVersion not defined

### ISSUE THREAD

**Issue by SirZach** [rwjblue/ember-cli-version-checker#6]
Not sure if this is right repo but after doing a fresh install of ember-cli, 0.2.0 or 0.2.7, the command line bombs with

```
zachmcgonigal@Zachs-MacBook-Pro:~/netuitive/git/ria$ ember --version
comparisonVersion is not defined
ReferenceError: comparisonVersion is not defined
    at Function.deprecatedAssertAbove (/Users/zachmcgonigal/netuitive/git/ria/node_modules/ember-cli-htmlbars/node_modules/ember-cli-version-checker/index.js:126:84)
    at Class.module.exports.init (/Users/zachmcgonigal/netuitive/git/ria/node_modules/ember-cli-htmlbars/ember-addon-main.js:12:13)
    at new Class (/Users/zachmcgonigal/netuitive/git/ria/node_modules/ember-cli/node_modules/core-object/core-object.js:18:12)
    at /Users/zachmcgonigal/netuitive/git/ria/node_modules/ember-cli/lib/models/addons-factory.js:44:19
    at visit (/Users/zachmcgonigal/netuitive/git/ria/node_modules/ember-cli/lib/utilities/DAG.js:23:3)
    at DAG.topsort (/Users/zachmcgonigal/netuitive/git/ria/node_modules/ember-cli/lib/utilities/DAG.js:82:7)
    at AddonsFactory.initializeAddons (/Users/zachmcgonigal/netuitive/git/ria/node_modules/ember-cli/lib/models/addons-factory.js:40:9)
    at Project.initializeAddons (/Users/zachmcgonigal/netuitive/git/ria/node_modules/ember-cli/lib/models/project.js:312:36)
    at Project.eachAddonCommand (/Users/zachmcgonigal/netuitive/git/ria/node_modules/ember-cli/lib/models/project.js:363:10)
    at module.exports (/Users/zachmcgonigal/netuitive/git/ria/node_modules/ember-cli/lib/cli/lookup-command.js:46:13)
```

**Comment by rwjblue** [comment]
Fixed by https://github.com/rwjblue/ember-cli-version-checker/pull/5 and released in 1.1.1.

Please `rm -rf node_modules && npm cache clear && npm install`.

**Comment by SirZach** [comment]
I'm not sure 911 responds that fast. Thanks!
